src/plugins/home/public/application/components/apm_fleet/index.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import {
+  EuiButton,
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingSpinner,
+  EuiPanel,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { getServices } from '../../kibana_services';
+
+const CentralizedContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+interface APIResponse {
+  hasData: boolean;
+}
+
+export function APMFleet() {
+  const { getBasePath } = getServices();
+  const basePath = getBasePath();
+  const [data, setData] = useState<APIResponse | undefined>();
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    async function fetchData() {
+      setIsLoading(true);
+      try {
+        const response = await fetch(`${basePath}/api/apm/fleet/hasData`);
```

Comment:
You should use the `http` client from core. Can't remember all the advantages but pretty sure it handles basepath for you:

```ts
core.http.get(`/api/apm/fleet/hasData`)
```